### PR TITLE
Save closing of splash-screen in the localstorage

### PR DESCRIPTION
**Pull request by arianegr**
In wave-core-new:

- add a new component  dialogs/dialog-splash-checkbox
- in user.service.ts  I added the two new methods to save/get the settings to/from the LocalStorage

in gfbio-app:
update  splash-dialog-component : add wave-dialog-splash-checkbox

### CODE REVIEW

**Comment by arianegr** [comment]
[UI: Schließen des Splash-Screens im LocalStorage des Browsers speichern](https://trello.com/c/8l28PReo/946-ui-schlie%C3%9Fen-des-splash-screens-im-localstorage-des-browsers-speichern)

File: projects/gfbio-app/src/app/splash-dialog/splash-dialog.component.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
     templateUrl: './splash-dialog.component.html',
     styleUrls: ['./splash-dialog.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class SplashDialogComponent {
-    // changeTick(event: MatCheckboxChange): void {
-    //     // TODO: remember user has clicked this
-    // }
+export class SplashDialogComponent implements OnInit {
+    splashName = 'showStartupSplashScreen';
```
Comment (ChristianBeilschmidt) [inline projects/gfbio-app/src/app/splash-dialog/splash-dialog.component.ts:10]: maybe you can make this a class constant and use it for loading the info…

File: projects/gfbio-app/src/app/app.component.ts [hunk 2]
```diff
@@ -240,7 +240,10 @@ export class AppComponent implements OnInit, AfterViewInit {
                     this.dialog.open(BasketDialogComponent, {data: {result}});
                 });
             } else {
-                this.dialog.open(SplashDialogComponent, {});
+                const showSplash = this.userService.getSettingFromLocalStorage('showStartupSplashScreen');
```
Comment (ChristianBeilschmidt) [inline projects/gfbio-app/src/app/app.component.ts:243]: …here, so that you are sure to always have the correct key.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]
I've one small comment.

File: projects/gfbio-app/src/app/app.component.ts (hunk 2 above)
Comment (arianegr) [reply projects/gfbio-app/src/app/app.component.ts]: [115f7bf](https://github.com/arianegr/geoengine-ui/commit/115f7bf06a2073675888913a3f4a1a5f89b0fdd2)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**